sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/TableChange.java

```diff
@@ -259,6 +298,64 @@ public int hashCode() {
     }
   }
 
+  interface ColumnPosition {
+    First FIRST = new First();
+
+    static ColumnPosition createAfter(String column) {
```

Comment:
I don't think `ColumnPosition` should assume the "create" operation. This can be used in column reordering as well. The parser supports it:

```
ALTER TABLE t ALTER COLUMN point.z AFTER point.y
```

Reply:
I don't think it's assuming create. I think it was meant to be "createAfter", like "createSparkSession". There was a previous comment asking it to rename it other than after...

Reply:
Or maybe I'm confused as well :P

Reply:
I think using `ColumnPosition.after` clears up the confusion.